src/app/cards/components/card-show-page/card-show-page.component.html

```diff
@@ -7,16 +7,16 @@
   <mat-card-content>
     <!-- Power and Toughness -->
     <div *ngIf="card.power != null && card.toughness != null">
-      <h4>Power / Toughness</h4>
-      {{card.power}} / {{card.toughness}}
+      <h3>Power / Toughness</h3>
+      <p>{{card.power}} / {{card.toughness}}</p>
     </div>
 
     <!-- Card Type -->
-    <h4>Card Type</h4>
+    <h3>Card Type</h3>
     <p>{{card.card_type}}</p>
 
     <!-- Description -->
-    <h4>Description</h4>
-    <p [innerHTML]="card.card_text | cardTextTransform"></p>
+    <h3>Description</h3>
+    <p *ngFor="let text of card.card_text | cardTextTransform" [innerHTML]="text"></p>
```

Comment:
`cardTextTransform` now returns an array of strings where each array is a separate line.

Also changed the `h4` headers to `h3` to give a bit more separation.